promise/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "js-interview-promise",
+  "version": "1.0.0",
+  "scripts": {
+    "promise": "tsx --watch promiseTimeout.ts"
```

Comment:
Why not `node --experimental-type-stripping promiseTimeout.ts`?

Reply:
because it's an experimental flag and we are not sure people during the interview would manage to set up nvm with the correct version as they probably stressed enough.